tests/models/mimi/test_modeling_mimi.py

```diff
@@ -411,8 +411,10 @@ def test_identity_shortcut(self):
 
     @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
     @require_torch_sdpa
-    @slow
     def test_eager_matches_sdpa_inference(self, torch_dtype: str):
```

Comment:
Can we add a comment at the top explaining why we need to overwrite this test in mimi? 🤗 

If the test is overwritten only to skip the case `torch_dtype == "float16" and torch_device == "cpu"`, can we delete the overwritten test and skip only that case?